Adding right columns, enabling column visibility and reordering

matheus-of-freitas opened the pull request:
# Column Visibility & Reorder
Adding the columns asked at the validation meeting, enabling column visibility (allowing to show and hide) and reordering (allowing to change the order of the columns).

matheus-of-freitas committed:
Adding right columns, enabling column visibility and reordering

glaubervila commented on backend/galaxy/views.py:
```diff
@@ -14,10 +14,10 @@
-    ordering_fields = ('id', 'megacube', 'mangaid', 'objra', 'objdec', 'harname', 'platetyp',)
-    ordering = ('id', 'megacube', 'mangaid', 'objra', 'objdec', 'harname', 'platetyp',)
+    filter_fields = ('id', 'megacube', 'mangaid', 'objra', 'objdec', 'nsa_iauname', 'mjdmed', 'exptime', 'airmsmed', 'seemed', 'nsa_z',)
+    search_fields = ('id', 'megacube', 'mangaid', 'objra', 'objdec', 'nsa_iauname', 'mjdmed', 'exptime', 'airmsmed', 'seemed', 'nsa_z',)
```
Testa o Search para esse monte de colunas, pq o search usa Like e deve ser usado só com colunas texto se uma dessas colunas for de outro tipo ele da erro.

matheus-of-freitas committed:
Update views.py

Wrong search_fields params. It makes a SQL "LIKE", so it would be problematic to include different data types other than strings.

glaubervila commented on backend/galaxy/views.py:
```diff
@@ -14,10 +14,10 @@
+    filter_fields = ('id', 'megacube', 'mangaid', 'objra', 'objdec', 'nsa_iauname', 'mjdmed', 'exptime', 'airmsmed', 'seemed', 'nsa_z',)
+    search_fields = ('megacube', 'nsa_iauname',)
+    ordering_fields = ('id', 'megacube', 'mangaid', 'objra', 'objdec', 'nsa_iauname', 'mjdmed', 'exptime', 'airmsmed', 'seemed', 'nsa_z',)
+    ordering = ('id', 'megacube', 'mangaid', 'objra', 'objdec', 'nsa_iauname', 'mjdmed', 'exptime', 'airmsmed', 'seemed', 'nsa_z',)
```
Ordering está errado tb só vi agora, 
o Ordering_filds é a lista de campos possiveis para se ordernar. ou que vão aceitar ordenação.
Já o Ordering = é qual campo de ordenação default, caso nenhuma seja passada.

matheus-of-freitas committed:
Update views.py

The "ordering" accepts the fields that will be ordered by default!
